runtime/src/enclave_rpc/session.rs

```diff
@@ -132,32 +133,36 @@ impl Session {
                 self.state = State::Handshake2(state);
             }
             State::Handshake2(mut state) => {
-                if state.is_initiator() {
-                    // <- e, ee, s, es
-                    let len = state.read_message(&data, &mut self.buf)?;
-                    let remote_static = state
-                        .get_remote_static()
-                        .expect("dh exchange just happened");
-                    self.info = self
-                        .verify_rak_binding(&self.buf[..len], remote_static)
-                        .await?;
+                // <- e, ee, s, es (initiator) | <- s, se (responder)
```

Comment:
```suggestion
```
Would remove as is distracting, or just mention that these were messages from handshake 1.